packages/suite/src/reducers/wallet/graphReducer.ts

```diff
@@ -6,7 +6,7 @@ import { GraphRange, GraphScale, AccountIdentifier, GraphData } from '@wallet-ty
 import { Action as SuiteAction } from '@suite-types';
 import { SETTINGS } from '@suite-config';
 
-interface State {
+export interface State {
```

Comment:
Every export I added anywhere is because TS complained, it's probably because I added `declaration` and `declarationMap` everywhere. Second option is needed because it will make work some additional editor features like Go To Definition etc.